macos/Info.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleIconFile</key>
+	<string>grabber.icns</string>
```

Comment:
I don't know much about MacOS, but isn't this file path `resources/grabber.icns`?
Or does the plist file automatically look in that folder?

Reply:
It auto looks there